files/gimp.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
```

Comment:
When an application, such as this one, has both a Flatpak and a Snap, I'd prefer to use the Flatpak.

https://flathub.org/apps/org.gimp.GIMP

We can install this during the first-time setup under productivity, too.

https://github.com/LukeShortCloud/winesapOS/blob/4.1.0-beta.1/scripts/winesapos-setup.sh#L457